apps/extension/client/testFixture/machine.ts

```diff
@@ -0,0 +1,10 @@
+import { createMachine } from "xstate";
```

Comment:
it might be good to add this to the repo and then gitignore it so updates to this wont be preserved in git. This is mainly useful (at the moment) as a kitchensink file for testing purposes.